build_plugin.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+rm prometheus/plugins/share/pluginPrometheus.jar
+
```

Comment:
The `bob.jar` I was using to build the extension was loading the existing `share/pluginPrometheus.jar` as a plugin and throwing an exception since it had a reference to the deleted `LuaBuilderPlugin` class.